lua/resession/init.lua

```diff
@@ -328,9 +330,10 @@ M.save_tab = function(name, opts)
   else
     tab_sessions[cur_tabpage] = nil
   end
+  require("resession.util").event("SaveTabPost")
```

Comment:
If we add an event payload that includes `is_tab_scoped`, then this event can also be `ResessionSavePost`